SonarQube.TeamBuild.PostProcessor/Program.cs

```diff
@@ -85,131 +59,5 @@ private static AnalysisConfig GetAnalysisConfig(TeamBuildSettings teamBuildSetti
-        {
-            logger.LogDebug(Resources.MSG_LoadingConfig, config.FileName);
-
-            switch (settings.BuildEnvironment)
-            {
-                case BuildEnvironment.LegacyTeamBuild:
-                    logger.LogDebug(Resources.SETTINGS_InLegacyTeamBuild);
-
-                    break;
-                case BuildEnvironment.TeamBuild:
-                    logger.LogDebug(Resources.SETTINGS_InTeamBuild);
-                    break;
-                case BuildEnvironment.NotTeamBuild:
-                    logger.LogDebug(Resources.SETTINGS_NotInTeamBuild);
-                    break;
-                default:
-                    break;
-            }
-
-            logger.LogDebug(Resources.SETTING_DumpSettings,
-                settings.AnalysisBaseDirectory,
-                settings.BuildDirectory,
-                settings.SonarBinDirectory,
-                settings.SonarConfigDirectory,
-                settings.SonarOutputDirectory,
-                settings.AnalysisConfigFilePath);
-        }
-
-        /// <summary>
-        /// Returns a boolean indicating whether the information in the environment variables
-        /// matches that in the analysis config file.
-        /// Used to detect invalid setups on the build agent.
-        /// </summary>
-        private static bool CheckEnvironmentConsistency(AnalysisConfig config, TeamBuildSettings settings, ILogger logger)
-        {
-            // Currently we're only checking that the build uris match as this is the most likely error
-            // - it probably means that an old analysis config file has been left behind somehow
-            // e.g. a build definition used to include analysis but has changed so that it is no
-            // longer an analysis build, but there is still an old analysis config on disc.
-
-            if (settings.BuildEnvironment == BuildEnvironment.NotTeamBuild)
-            {
-                return true;
-            }
-
-            string configUri = config.GetBuildUri();
-            string environmentUi = settings.BuildUri;
-
-            if (!string.Equals(configUri, environmentUi, System.StringComparison.OrdinalIgnoreCase))
-            {
-                logger.LogError(Resources.ERROR_BuildUrisDontMatch, environmentUi, configUri, settings.AnalysisConfigFilePath);
-                return false;
-            }
-
-            return true;
-        }
-
-        private static ProjectInfoAnalysisResult InvokeSonarRunner(AnalysisConfig config, ConsoleLogger logger)
-        {
-            ISonarRunner runner = new SonarRunnerWrapper();
-            logger.IncludeTimestamp = false;
-            ProjectInfoAnalysisResult result = runner.Execute(config, Enumerable.Empty<string>() /* todo */, logger);
-            logger.IncludeTimestamp = true;
-            return result;
-        }
-
-        private static void UpdateTeamBuildSummary(AnalysisConfig config, ProjectInfoAnalysisResult result, ILogger logger)
```

Comment:
This method is obsolete. The code was moved to the SummaryReportBuilder some time ago.